src/index.js

```diff
@@ -99,6 +99,19 @@ setInterval(() => {
   // the metrics data collected above.
   data = {
     counters: {
+      "node.eventloop.latency.0.milliseconds": latencies['0'] || 0,
```

Comment:
Can I suggest `ms` instead of full "milliseconds"?

Reply:
Ah, I had that but it didn't make it into the commit